docs/source/platform/schema-validation.md

```diff
@@ -1,60 +1,101 @@
+When metrics are sent to a service, the changes are verified in the following manner:
 
-To improve the understanding of field usage within an API, Apollo Server extends GraphQL with rich tracing data that demonstrates _how_ a GraphQL field is used and _when_ it's safe to change or eliminate a field.
+- **Addition**: Automatically marked as a `Notice`
+- **Update**: Automatically marked as a `Warning`
+- **Removal**: If an operation uses the removed type, field, enum, argument, or other schema element, marked as `Failure`
 
-> For details on how tracing data can be used to avoid shipping breaking changes to clients, check out the schema history tooling in [Apollo Engine](https://www.apollographql.com/platform) which utilizes actual usage data to provide warnings and notices about changes that might break existing clients.
+The overall schema validation check will fail if there are any changes marked as a `Failure`.
 
-Since GraphQL clients only receive exactly what they ask for, adding new fields, arguments, queries, or mutations won't introduce any new breaking changes and these changes can be confidently made without consideration about existing clients or field usage metrics.
+> Occasionally a service or schema tag does not contain any metrics. In this case, schema validation does cannot verify changes, so removals are automatically set as `Failure`. Additionally warnings will fail Github status checks.
+
+<h2 id="performing-changes">Strategies for performing schema changes</h2>
+
+Defining strategies to perform these changes with minimal impact on clients enables maintainable evolving a schema in response to rapidly changing product requirements. The insight necessary for these techniques is adding new fields, arguments, queries, or mutations won't introduce any new breaking changes. These changes can be confidently made without consideration about existing clients or field usage metrics, since GraphQL clients receive exactly what they ask for.
 
 _Field rollover_ is a term given to an API change that's an evolution of a field, such as a rename or a change in arguments. Some of these changes can be really small, resulting in many variations and making an API harder to manage.
 
 We'll go over these two kinds of field rollovers separately and show how to make these changes safely.
 
 <h3 id="renaming-or-removing">Renaming or removing a field</h3>
 
-When a field is unused, renaming or removing it is as straightforward as it sounds: it can be renamed or removed.  Unfortunately, if a GraphQL deployment doesn't have per-field usage metrics, additional considerations should be made.
+When a field is unused, renaming or removing can be performed immediately without affecting clients. Unfortunately, additional considerations should be made if a client uses the field or a GraphQL deployment doesn't have per-field usage metrics.
```

Comment:
Maybe a warning? While this can make sense during active development, developers should be very cautious about modifying any fields that are present on the schema running in production.